packages/hdwallet-ledger/src/currencies.ts

```diff
@@ -0,0 +1,22 @@
+// Taken from https://github.com/LedgerHQ/ledgerjs/blob/master/packages/cryptoassets/src/currencies.ts#L299
```

Comment:
Very much stripped down version of the above - kept the `currencies` name to match upstream